providers/kit/metadata_test.go

```diff
@@ -0,0 +1,224 @@
+// nolint
+package kit
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/amp-labs/connectors"
+	"github.com/amp-labs/connectors/common"
+	"github.com/amp-labs/connectors/test/utils/mockutils"
+	"github.com/amp-labs/connectors/test/utils/mockutils/mockcond"
+	"github.com/amp-labs/connectors/test/utils/mockutils/mockserver"
+	"github.com/amp-labs/connectors/test/utils/testroutines"
+	"github.com/amp-labs/connectors/test/utils/testutils"
+)
+
+func TestListObjectMetadata(t *testing.T) { // nolint:funlen,gocognit,cyclop
+	t.Parallel()
+
+	broadcastsresponse := testutils.DataFromFile(t, "broadcasts.json")
+	Customfieldsresponse := testutils.DataFromFile(t, "custom_fields.json")
```

Comment:
lower camel case prefered.
For consistency it should be at least `customfieldsresponse`